src/components/Vote.jsx

```diff
@@ -0,0 +1,24 @@
+export default function VoteOnReview({ review }) {
+
+    const voteOnReview = () => {
+
+    let newScore = {
+        inc_votes: 1
+     }
+        const requestOptions = {
+            method: 'PATCH',
+            headers: { 'Content-Type': 'application/json' },
+            body: JSON.stringify(newScore)
+        }
+        voteFunction(requestOptions)
+    }
+
+    function voteFunction(requestOptions) {
+        fetch(`https://final-boardgame-api.herokuapp.com/api/reviews/${review.review_id}`, requestOptions)
+            .then((response) => response.json()).then()
+    }
+
+    return (
+        <button className="buttons" onClick={() => voteOnReview()}>Vote</button>
+    )
```

Comment:
When i click on the vote button I want to see some optimistical rendering, aka seeing the vote go up live rather than needing to refresh the page. It would be cool for the vote button and the score to be more close together as well, just so it's obvious that by clicking on vote you are adding one to the score, right now it's not too clear that is happening.